stdlib/2and3/keyword.pyi

```diff
@@ -0,0 +1,6 @@
+# Stubs for keyword
+
+from typing import Sequence
+
+def iskeyword(s: str) -> bool: ...
```

Comment:
Agreed. But we should at least allow `unicode` (on Python 2) and `bytes` (on Python 3).